shopsdk/config.json

```diff
@@ -0,0 +1,117 @@
+  "digital_sentry_url": "status.domain.tld",
+  "nwcs_sentry_url": "status.domain.tld",
+  "accf_sentry_url": "status.domain.tld",
+  "printful_cdn_url": "status.domain.tld",
+  "pwinty_cdn_url": "status.domain.tld",
+  "accf_cdn_url": "status.domain.tld",
+  "oc_url_root": "status.domain.tld",
+  "datadog_api_key": "pleasechangeme",
+  "pwinty_app_key": "pleasechangeme",
+  "photos_app_key": "pleasechangeme",
+  "printful_app_key": "pleasechangeme",
+  "digital_app_key": "pleasechangeme",
+  "pwinty_api_key": "pleasechangeme",
+  "printful_api_key": "pleasechangeme",
+  "oc_url_hash": "pleasechangeme",
+  "main_oc_username_hash": "pleasechangeme",
+  "main_oc_encryption_hash": "pleasechangeme",
+  "main_oc_directory_hash": "pleasechangeme",
+  "main_oc_primary_file_hash": "pleasechangeme",
+  "main_oc_secondary_file_hash": "pleasechangeme",
+  "main_oc_password_hash": "pleasechangeme",
+  "main_oc_primary_local_hash": "pleasechangeme",
+  "main_oc_secondary_local_hash": "pleasechangeme",
+  "secn_oc_username_hash": "pleasechangeme",
+  "secn_oc_encryption_hash": "pleasechangeme",
+  "secn_oc_directory_hash": "pleasechangeme",
+  "secn_oc_primary_file_hash": "pleasechangeme",
+  "secn_oc_secondary_file_hash": "pleasechangeme",
+  "secn_oc_password_hash": "pleasechangeme",
+  "secn_oc_primary_local_hash": "pleasechangeme",
+  "secn_oc_secondary_local_hash": "pleasechangeme",
+  "options_oc_username_hash": "pleasechangeme",
+  "options_oc_encryption_hash": "pleasechangeme",
+  "options_oc_directory_hash": "pleasechangeme",
+  "options_oc_primary_file_hash": "pleasechangeme",
+  "options_oc_secondary_file_hash": "pleasechangeme",
+  "options_oc_password_hash": "pleasechangeme",
+  "options_oc_primary_local_hash": "pleasechangeme",
+  "options_oc_secondary_local_hash": "pleasechangeme",
+  "othr_oc_username_hash": "pleasechangeme",
+  "othr_oc_encryption_hash": "pleasechangeme",
+  "othr_oc_directory_hash": "pleasechangeme",
+  "othr_oc_primary_file_hash": "pleasechangeme",
+  "othr_oc_secondary_file_hash": "pleasechangeme",
+  "othr_oc_password_hash": "pleasechangeme",
+  "othr_oc_primary_local_hash": "pleasechangeme",
+  "othr_oc_secondary_local_hash": "pleasechangeme",
+  "accf_oc_username_hash": "pleasechangeme",
+  "accf_oc_encryption_hash": "pleasechangeme",
+  "accf_oc_directory_hash": "pleasechangeme",
+  "accf_oc_primary_file_hash": "pleasechangeme",
+  "accf_oc_secondary_file_hash": "pleasechangeme",
+  "accf_oc_password_hash": "pleasechangeme",
+  "accf_oc_primary_local_hash": "pleasechangeme",
+  "accf_oc_secondary_local_hash": "pleasechangeme",
+  "nwcs_oc_username_hash": "pleasechangeme",
+  "nwcs_oc_encryption_hash": "pleasechangeme",
+  "nwcs_oc_directory_hash": "pleasechangeme",
+  "nwcs_oc_primary_file_hash": "pleasechangeme",
+  "nwcs_oc_secondary_file_hash": "pleasechangeme",
+  "nwcs_oc_password_hash": "pleasechangeme",
+  "nwcs_oc_primary_local_hash": "pleasechangeme",
+  "nwcs_oc_secondary_local_hash": "pleasechangeme"
+}
```

Comment:
Oh my God why so many names

Reply:
Yeah... that is why I scrapped encryption, it was too much of a PITA.